docs/security/authorization-pushdown.md

```diff
@@ -43,6 +43,36 @@ authorization.ranger.password=PWD123
 authorization.ranger.service.name=hiveRepo
 ```
 
+### Authorization Iceberg with Ranger properties
+
+In order to use the Authorization Ranger Iceberg Plugin, you need to configure the following properties and [Lakehouse_Iceberg catalog properties](../lakehouse-iceberg-catalog.md#catalog-properties):
+
+| Property Name                       | Description                                                                                                                                          | Default Value | Required | Since Version    |
+|-------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|----------|------------------|
+| `authorization-provider`            | Providers to use to implement authorization plugin such as `ranger`.                                                                                 | (none)        | No       | 0.6.0-incubating |
+| `authorization.ranger.admin.url`    | The Apache Ranger web URIs.                                                                                                                          | (none)        | No       | 0.6.0-incubating |
+| `authorization.ranger.auth.type`    | The Apache Ranger authentication type `simple` or `kerberos`.                                                                                        | `simple`      | No       | 0.6.0-incubating |
+| `authorization.ranger.username`     | The Apache Ranger admin web login username (auth type=simple), or kerberos principal(auth type=kerberos), Need have Ranger administrator permission. | (none)        | No       | 0.6.0-incubating |
+| `authorization.ranger.password`     | The Apache Ranger admin web login user password (auth type=simple), or path of the keytab file(auth type=kerberos)                                   | (none)        | No       | 0.6.0-incubating |
+| `authorization.ranger.service.name` | The Apache Ranger service name.                                                                                                                      | (none)        | No       | 0.6.0-incubating |
```

Comment:
Maybe need to change `0.6.0-incubating` to `0.8.0-xxx`?

Reply:
This option reuse Hive properties. I don't a a new properity. This propeties are introduced in 0.6.0-incubating.

Reply:
Change to 0.8.0-incubating.